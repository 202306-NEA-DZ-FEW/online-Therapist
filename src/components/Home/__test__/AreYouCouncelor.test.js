import renderer from "react-test-renderer";

import AreYouCounselor from "@/components/Home/AreYouCouncelor";

it("renders correctly", () => {
    const tree = renderer.create(<AreYouCounselor />).toJSON();
    expect(tree).toMatchSnapshot();
});
