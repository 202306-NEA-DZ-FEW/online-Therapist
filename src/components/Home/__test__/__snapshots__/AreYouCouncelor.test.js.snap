// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="font-atkinson mt-24"
>
  <div
    style={
      Object {
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "opacity": 0,
          "transform": "translateY(75px) translateZ(0)",
        }
      }
    >
      <h2
        className="text-center lg:text-start font-bold block text-2xl md:text-4xl rtl:md:text-3xl  rtl:lg:text-4xl uppercase break-words"
      >
        counselor.heading
      </h2>
    </div>
  </div>
  <div
    style={
      Object {
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "opacity": 0,
          "transform": "translateY(75px) translateZ(0)",
        }
      }
    >
      <p
        className="uppercase text-center lg:text-start text-lg md:text-xl text-black/70 my-2"
      >
        counselor.subHeading
      </p>
    </div>
  </div>
  <div
    className="bg-white/95 flex flex-col lg:flex-row items-center justify-center text-start  rounded-md  shadow-2xl min-h-[20rem]"
  >
    <div
      style={
        Object {
          "overflow": "hidden",
          "position": "relative",
        }
      }
    >
      <div
        style={
          Object {
            "opacity": 0,
            "transform": "translateY(75px) translateZ(0)",
          }
        }
      >
        <div
          className="p-16 space-y-4 lg:p-0 lg:space-y-0 lg:flex lg:flex-col lg:items-center lg:justify-center"
        >
          <span
            className="text-md md:text-lg p-12 font-atkinson text-black/80"
          >
            counselor.paragraph
          </span>
          <div
            className="flex items-center justify-center "
          >
            <a
              href="/requirements"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              <div
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "opacity": 0,
                      "transform": "translateY(75px) translateZ(0)",
                    }
                  }
                >
                  <div
                    className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-48 h-14 bg-white hover:cursor-pointer"
                  >
                    <span
                      className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 translate-x-full bg-Teal group-hover:translate-x-0 ease"
                    >
                      <svg
                        className="w-6 h-6 rotate-180"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M14 5l7 7m0 0l-7 7m7-7H3"
                          strokeLinecap="round"
                          strokeLinejoin="round"
                          strokeWidth="2"
                        />
                      </svg>
                    </span>
                    <span
                      className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:-translate-x-full ease"
                    >
                      counselor.learnMore
                    </span>
                    <span
                      className="relative invisible"
                    >
                      $
                      counselor.learnMore
                    </span>
                  </div>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <img
      className="hidden md:block w-96"
      src="/Images/therapist.jpg"
    />
  </div>
</div>
`;
